src/foundry/common/grid/base.d.mts

```diff
@@ -551,10 +552,36 @@ declare namespace BaseGrid {
     j: number;
   }
 
+  /**
+   * 2D offset coordinates of a grid space.
+   */
+  interface Offset2D {
+    /** The row coordinate (an integer) */
+    i: number;
+
+    /** The column coordinate (an integer) */
+    j: number;
+  }
+
+  /**
+   * 3D offset coordinates of a grid space.
+   */
+  interface Offset3D extends Offset {
+    /**
+     * The vertical coordinate (an integer)
+     */
+    k: number;
+  }
+
   type OffsetRange = [i0: number, j0: number, i1: number, j1: number];
 
+  // TODO: Figure out to do with this lil guy
```

Comment:
What's up with this? I vaguely recall talking about deprecating it or something?